source/ui/Components/Memory/MemoryComponent.qml

```diff
@@ -81,17 +84,44 @@ Item {
         }
     }
 
+
+    Component {
+        id: rowdelegate
+        // collapse row if cell is not needed
+        Rectangle{
+            height: (styleData.row % (number_bits / 8) == 0) ? 25 : 0
+        }
+    }
+
+    Component {
+        // disable row when cell is not needed
+        id: itemdelegate
+        Text {
+            visible: (styleData.row % (number_bits / 8) == 0) ? true : false
+            enabled: (styleData.row % (number_bits / 8) == 0) ? true : false
+            text: styleData.value
+            verticalAlignment: Text.AlignVCenter
+        }
+    }
+
     Component {
         // makes each memory cell editable by using a textbox
         // when editing is finished the new value is passed to the memory in the core
         id: inputBox
+
         TextField {
             id: textFieldMemoryValue
-            text: styleData.value
+            text: (styleData.row % (number_bits / 8) == 0) ? styleData.value : ""
+            // hide input if cell is not needed
+            visible: (styleData.row % (number_bits / 8) == 0) ? true : false
+            enabled: (styleData.row % (number_bits / 8) == 0) ? true : false
 
```

Comment:
Add `import QtQuick.Controls.Styles 1.4` at the top (if not there already) and add
```
style: TextFieldStyle: {
    renderType: Text.QtRendering
    background: Rectangle {
                    id: styleRec
                    color: "transparent"
    }
}
```
to remove strange rendering artifacts and the textfield outline (The background part is taken from your old memory component).